MinnPost/object-sync-for-salesforce
# 441 mapping wordpress id fields

## What does this Pull Request do?

This fixes #441 by making WordPress ID fields uneditable, by default, by this plugin. It provides a filter to change this, but in general ID fields should not be editable and this sets that behavior to prevent confusion.

## How do I test this Pull Request?

- map an ID field
- make sure the value doesn't get synced
- use the filter to override that
